app/sources/DataInspector.m

```diff
@@ -249,6 +249,17 @@ static long double ieeeToLD(const void *bytes, unsigned exp, unsigned man) {
 
 static id floatingPointDescription(const unsigned char *bytes, NSUInteger length, enum Endianness_t endianness) {
     switch (length) {
+        case sizeof(short):
```

Comment:
I think `short` here should be `uint16_t`.

I'll update the others too so it's not mixing types.